JSDemos/Demos/Toolbar/Multiline/jQuery/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <title>DevExtreme Demo</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
+    <script src="../../../../../node_modules/jquery/dist/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="../../../../../node_modules/devextreme/dist/css/dx.common.css" />
+    <link rel="stylesheet" type="text/css" href="../../../../../node_modules/devextreme/dist/css/dx.light.css" />
+    <script src="../../../../../node_modules/devextreme/dist/js/dx.all.js"></script>
+    <script src="data.js"></script>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <script src="index.js"></script>
+  </head>
+  <body class="dx-viewport">
+    <div class="demo-container">
+      <div id="resizable-container" class="resizable-container">
+        <div id="toolbar"></div>
+      </div>
+      <div class="options-container">
+        <div class="caption">Options</div>
+        <div id="toolbarModeRadioGroup"></div>
```

Comment:
It's better to rename it 'toolbar-modes' and need check the naming of id attributes. I see that <div id="resizable-container"> using with dash.

Reply:
Why `resizable-container`? It's about option block

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/0dab880278ac961d41ee713b6717c3b6fb302ced